string map_file_name
---
uint8 code
string message
